src/app/pages/Languages.svelte

```diff
@@ -23,8 +28,21 @@
     sources: [],
   };
 
-  onMount(async () => {
+  async function getLanguageSources() {
     languageInformation.sources = await worker.getFilesFromStorage();
+  }
+
+  // listen to changes to the package compilation and recalculate word count
+  compileSuccess.subscribe((currentValue) => {
+    downloadReady = currentValue;
+    if (currentValue) {
+      getLanguageSources();
+    }
+  });
```

Comment:
As mentioned in these docs <https://svelte.dev/tutorial/auto-subscriptions>, there is a sublte bug here; `.subscribe()` returns an `unsubscribe()` function that must be called in the `onDestroy()` hook; however, if you use the `$compileSuccess` syntax, Svelte will automatically subscribe and unsubscribe for you.

```suggestion
  $: downloadReady = $compileSuccess;
```